source/_docs/free-https.md

```diff
@@ -0,0 +1,205 @@
+      </li>
+      <li>Visit the module configuration page (<code>/admin/config/system/hsts</code>).</li>
+      <li>Check the <strong>Enable HTTP Strict Transport Security</strong> checkbox, set <strong>Max Age</strong> to at least <strong>1 year</strong> and click <strong>Save Configuration</strong>.</li>
+    </ol>
+    <p>Once installed and configured, the following header will be sent in responses:</p>
+    <pre><code>strict-transport-security: max-age=31536000</code></pre>
+  </div>
+</div>
+
+## Compare to Legacy HTTPS
+<table class="table  table-bordered table-responsive">
+  <thead>
+    <tr>
+      <th></th>
+      <th>Legacy</th>
+      <th>Free & Automated</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>Price</th>
+      <td>$30/mo per environment</td>
+      <td>Free for all environments</td>
+    </tr>
+    <tr>
+      <th>Certificate type</th>
+      <td>Bring your own</td>
+      <td>Shared, issued by Let's Encrypt</td>
+    </tr>
+    <tr>
+      <th>Renewal</th>
+      <td>Manual</td>
+      <td>Automatic</td>
+    </tr>
+    <tr>
+      <th>Unique inbound IP Address</th>
+      <td>Yes</td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <th>Client Support</th>
+      <td>94.58% of browsers<br />Some very old browsers not supported
+      <sup><a href="http://caniuse.com/#search=TLS%201.1">1</a></sup></td>
+      <td>94.33% of browsers<br />Some very old browsers not supported
+      <sup><a href="http://caniuse.com/#search=TLS%201.2">1 <a href="http://caniuse.com/#search=SNI">2</a></sup></td>
+    </tr>
+    <tr>
+      <th><a href="https://www.ssllabs.com/ssltest/">SSL Labs</a> Rating</th>
+      <td>A</td>
+      <td>A+ <a href="#http-strict-transport-security-header">with HSTS</a></td>
+    </tr>
+    <tr>
+      <th>Protocol</th>
+      <td>TLS 1.1 & 1.2</td>
+      <td>TLS 1.2 only</td>
+    </tr>
+    <tr>
+      <th>Ciphers</th>
+      <td>Weak 3DES cipher</td>
+      <td>No 3DES cipher</td>
+    </tr>
+    <tr>
+      <th>Delivery</th>
+      <td>ORD Datacenter</td>
+      <td>Global CDN</td>
+    </tr>
+  </tbody>
+</table>
+
+## Frequently Asked Questions
+
+### Does upgrading involve HTTPS interruptions or downtime?
+No, after you update your DNS records, traffic will gracefully switch over and involves no downtime or HTTPS interruption.
+
+**Caveat:** If after upgrading you add a new domain that is not already routed to Pantheon, then it will take 25 minutes to an hour for HTTPS to be ready for that new domain. Pre-provisioning HTTPS for new domains is planned after early access, with the full release.
+
+### Which browsers and operating systems are supported?
+All modern browsers and operating systems are supported. For details, see the **Handshake Simulation** portion of this [report](https://www.ssllabs.com/ssltest/analyze.html?d=pantheon.io).
+
+### When do I stopped being billed $30/month?
+Pantheon will remove legacy load balancers and stop billing within 30 days of upgrading.
```

Comment:
Can we clarify this a little?

Once we stop charging (i.e. the LBs are deleted) we won't be able to automatically revert the customer. Do we want to say "after 30 days of upgrading"? Or "within 60 days, with the ability to revert for 30 days" ?